pkg/localstore/gc.go

```diff
@@ -165,13 +162,21 @@ func (db *DB) collectGarbage() (collectedCount uint64, done bool, err error) {
 		return 0, false, err
 	}
 
+	var totalChunksEvicted uint64
+
 	// get rid of dirty entries
 	for _, item := range candidates {
 		if swarm.NewAddress(item.Address).MemberOf(db.dirtyAddresses) {
-			collectedCount--
 			continue
 		}
 
+		if gcSize-totalChunksEvicted <= target {
```

Comment:
I am not able to understand this check. I see the comment on the function `gcTarget` needs to be fixed?

Reply:
the comment has been fixed, thanks.
we also added another comment over this check that explains its purpose.